### ISSUE TITLE
Update state_get_account_info request type

### ISSUE THREAD

**Issue by RitaMAllenCA** [casper-network/casper-node#4204]
This RPC should support taking an AccountHash or PublicKey as the account ID

blocks: https://github.com/casper-ecosystem/casper-client-rs/issues/55

**Comment by RitaMAllenCA** [comment]
duplicate of #4203
